.circleci/config.yml

```diff
@@ -59,6 +59,66 @@ jobs:
           name: Post Test Results to GitHub
           command: bundle exec danger || echo "danger failed"
           when: always # Run this even when tests fail
+
+  "Execute tests on macOS (Xcode 11.0.0, Ruby 2.5)":
+    macos:
+      xcode: "11.0.0"
+    environment:
+      CIRCLE_TEST_REPORTS: ~/test-reports
+      LC_ALL: en_US.UTF-8
+      LANG: en_US.UTF-8
+    shell: /bin/bash --login -eo pipefail
+    steps:
+      - checkout
+
+      - restore_cache:
+          key: v1-macos-gems-{{ checksum "Gemfile" }}-{{ checksum "fastlane.gemspec" }}
+      - restore_cache:
+          keys:
+            - v3-homebrew-{{ epoch }}
+            - v3-homebrew
```

Comment:
Are the cached artifacts the same for builds with Xcode 11.0.0 and Xcode 9.0.1? 🤔 It looks like we are using the same key for the cache.

Reply:
Doe these matter regarding homebrew? I kinda assumed the Xcode version doesn't.